torch/nn/functional.py

```diff
@@ -1206,8 +1206,8 @@ def batch_norm(input, running_mean, running_var, weight=None, bias=None,
     )
 
 
-def instance_norm(input, running_mean, running_var, weight=None, bias=None,
-                  use_input_stats=True, momentum=0.1, eps=1e-5):
+def instance_norm(input, running_mean=None, running_var=None, weight=None,
```

Comment:
Running stats should really be optional here as they are not typically used in IN